stem/stem.py

```diff
@@ -285,6 +285,15 @@ def __transfer_vtk_files_to_main_output_directories(self):
                         else:
                             main_vtk_output_dirs[part_name] = Path(self.input_files_dir) / output_settings.output_dir
                     else:
+                        # create main output directory if it does not exist
+                        if not (os.path.isdir(main_vtk_output_dirs[part_name])):
+                            if not (os.path.isdir(Path(self.input_files_dir) / output_settings.output_dir)):
+                                raise Exception(
```

Comment:
actually, I dont think an exception should be raised at all in case there is no output. 

The process should just continue

Reply:
furthermore, this check assumes a relative path, I think it will fail in case of an absolute path

Reply:
Ok will change it to just a print statement